internal/slabutil/mock.go

```diff
@@ -31,6 +31,8 @@ func (h *MockHost) DownloadSector(w io.Writer, root consensus.Hash256, offset, l
 	sector, ok := h.sectors[root]
 	if !ok {
 		return errors.New("unknown root")
+	} else if uint64(offset)+uint64(length) > rhpv2.SectorSize {
+		return errors.New("offset+length out of bounds")
```

Comment:
might be worth doing `fmt.Errorf("out of bounds, %v+%v > %v", offset, lengt, rhpv2.SectorSize")`?

Reply:
meh, this is just a mock anyway, and it's not exactly hard to debug if something does go wrong